cosmo_tester/test_suites/scale_tests/blueprints_and_deployments_scale_test.py

```diff
@@ -0,0 +1,147 @@
+########
+# Copyright (c) 2014 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+import uuid
+from retrying import retry
+from datetime import datetime
+from multiprocessing import Process
+from cosmo_tester.test_suites.snapshots import snapshot_util
+from manager_rest.storage.models_states import ExecutionState
+from cosmo_tester.framework.fixtures import image_based_manager
+from cosmo_tester.framework.fixtures import hello_world  # noqa
+
+manager = image_based_manager
+DEPLOYMENTS_COUNT = 10
+PROCESS_COUNT = 10
+
+
+def test_hello_world_deployments_scale(             # noqa
+        manager, hello_world, attributes, logger):
+    hello_world.inputs.update({
+        'agent_user': attributes.centos7_username,
+        'image': attributes.centos7_image_name,
+    })
+
+    hello_world.clone_example()
+    _assert_create_deployments(manager.client, hello_world, logger)
+    _waiting_for_active_executions(manager.client)
+    _assert_create_snapshot(manager.client)
+    _check_response_time(manager.client, logger)
+    _assert_delete_deployments(manager.client, hello_world.name, logger)
+
+
+def _assert_create_deployments(client, blueprint_example, logger):
```

Comment:
get DEPLOYMENTS_COUNT, PROCESS_COUNT as input
and some comments about the test..